pandas/tests/scalar/test_timedelta.py

```diff
@@ -128,6 +128,61 @@ def test_unary_ops(self):
         assert abs(-td) == td
         assert abs(-td) == Timedelta('10d')
 
+    def test_mul(self):
+        # GH#19365
+        td = Timedelta(minutes=3)
+
+        result = td * 2
+        assert result == Timedelta(minutes=6)
+
+        result = td * np.int64(1)
+        assert isinstance(result, Timedelta)
+        assert result == td
+
+        result = td * 1.5
+        assert result == Timedelta(minutes=4, seconds=30)
+
+        result = td * np.array([3, 4], dtype='int64')
+        expected = np.array([9, 12], dtype='m8[m]').astype('m8[ns]')
+        tm.assert_numpy_array_equal(result, expected)
+
+        with pytest.raises(TypeError):
+            td * pd.Timestamp(2016, 1, 2)
+
+    def test_add_datetimelike(self):
+        # GH#19365
+        td = Timedelta(10, unit='d')
+
+        result = td + datetime(2016, 1, 1)
+        assert result == pd.Timestamp(2016, 1, 11)
+
+        result = td + pd.Timestamp('2018-01-12 18:09')
+        assert result == pd.Timestamp('2018-01-22 18:09')
+
+        result = td + np.datetime64('2018-01-12')
+        assert result == pd.Timestamp('2018-01-22')
+
+    @pytest.mark.parametrize('op', [lambda x, y: x + y,
+                                    lambda x, y: y + x])
+    def test_add_timedeltalike(self, op):
```

Comment:
these are also duplicative. pls consolidate.

Reply:
With what?

Reply:
tests_ops_offset. you can prob just remove that. but pls look for existing tests and modify / remove them.

Reply:
Done.  There's some more organizing of Timedelta arith tests I'll do in the next pass.  These need to be much more thorough.